src/tools/dev/masonry/opts/mb-3.4.1-darwin-21-x86_64-release.json

```diff
@@ -0,0 +1,52 @@
+{"bootstrap_visit":
+ {"version": "3.4.1",
+ "build_types": ["release"],
+ "branch": "3.4RC",
+ "arch":   "darwin-x86_64",
+ "cert":   "Developer ID Application: Lawrence Livermore National Laboratory (A827VH86QR)",
+ "entitlements": "/Users/miller86/visit/visit/34rc/src/tools/dev/masonry/opts/visit.entitlements",
+ "notarize": {"username":"miller86@llnl.gov",
+              "password":"@keychain:VisIt",
+              "asc_provider":"A827VH86QR",
+              "bundle_id":"gov.llnl.visit"},
+ "make_nthreads": 2,
+ "skip_checkout": "yes",
+ "git": {"mode":"ssh","git_uname":"markcmiller86"},
+ "build_visit":  { "cmake_ver": "3.24.3",
+                   "args":"--no-thirdparty",
+                   "libs":["cmake",
+                       "vtkm",
+                       "ispc",
+                       "embree",
+                       "tbb",
+                       "ospray",
+                       "python",
+                       "vtk",
```

Comment:
Do I need both "vtk" and "vtk9" here. I think just "vtk9", rigt?

Reply:
correct, just vtk9